src/checkpoint/checkpoint_txn.c

```diff
@@ -2040,6 +2040,18 @@ __checkpoint_lock_dirty_tree(
             is_drop = true;
     }
 
+    /*
+     * Force checkpoint for original files when using a live restore file system. Live restore needs
+     * to write metadata for each file being restored, but it doesn't dirty original btree's during
+     * background migration. By forcing checkpoints we ensure that all original btrees will have
+     * their metadata updated.
+     *
+     * Alternatively, we could perform a forced checkpoint on completion of migration for every file
+     * migrated. However, that would be an expensive operation.
+     */
+    if (F_ISSET(S2C(session), WT_CONN_LIVE_RESTORE_FS) && btree->original)
```

Comment:
I don't think this is necessary anymore if we're okay with live restore potentially losing the data copied from an original file. That content should just be the header and copying it multiple times should not be a problem. Let me know what you think here.

Reply:
I'm happy to drop this and just rewrite the original files. I suspect that's what we do already if the original file is closed before a checkpoint since we don't have forcing logic in `__wt_checkpoint_close` anymore.

Reply:
discussed with @ajmorton offline, will leave this as is for now.